crates/tools/python/README.md

```diff
@@ -13,4 +13,3 @@ conda env create -f env.yml
 conda activate hardhat
 jupyter notebook --NotebookApp.iopub_data_rate_limit=1.0e10
 ```
-
```

Comment:
Is this removing a line ending at end-of-file? Those should be retained imo

See https://stackoverflow.com/a/5813359 for reasoning

Reply:
There were two lines, prettier just removed the extra one